src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IVsEnvironmentServices.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal interface IVsEnvironmentServices
```

Comment:
ya , i had a feeling about that as well . does IEnvironementService sounds good i was not very happy with that as well ? and i can create IRenamer to do things specific to rename.